rosidl_typesupport_opendds_cpp/rosidl_typesupport_opendds_cpp/__init__.py

```diff
@@ -53,12 +55,29 @@ def generate_dds_opendds_cpp(
         except FileExistsError:
             pass
 
+        rpc_file_path = str(Path(output_basepath).parents[3]) + '/src/rosidl_typesupport_opendds/rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp'
+        rpc_file = str(rpc_file_path) + '/RPC.idl'
+
+        # Compile RPC.idl
+        if not RPC_compiled :
+          try:
+              cmd = [idl_pp, rpc_file, "-Lc++11", "-o", output_path, "-Cw"]
+              for include_dir in include_dirs:
+                  cmd += ['-I', include_dir]
+              cmd += ['-I', rpc_file_path]
+              subprocess.check_call(cmd)
```

Comment:
This needs to move to the CMake build process.  We don't need to change it right now, but let's keep track of it for later.

Reply:
Added 'TODO' comment